milestone3/frontend/src/components/RecipePage.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { ChangeEvent, useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom'
+import recipes from '../recipeData';
+import './RecipePage.css'
+
+interface RecipePageProps {
+    external: boolean;
+}
+
+export default function RecipePage(props: RecipePageProps) {
+    const { name, id } = useParams();
+    const defaultRecipe = recipes[0];
+    const [recipe, setRecipe] = useState(defaultRecipe)
+    const [newIngredient, setNewIngredient] = useState('');
+    const [allIngredients, setAllIngredients] = useState(defaultRecipe.ingredients)
+    const [newInstruction, setNewInstruction] = useState('');
+    const [allInstructions, setAllInstructions] = useState(defaultRecipe.instructions)
```

Comment:
So we don't want to set our ingredients and instructions to be from the defaultRecipe. If you look at your pages, this is causing all of your recipes to have the same ingredients and instructions list as the first recipe! 
To fix this, you can call the setAllIngredients and setAllInstructions functions with the recipe we get back in our useEffect. This way the instructions and ingredients lists will reflect the current recipe on this page